api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -789,6 +777,49 @@ public void purgeVisit_shouldFailIfTheVisitHasEncountersAssociatedToIt() {
 		Context.getVisitService().purgeVisit(visit);
 	}
 	
+	/**
+	 * @see VisitService#purgeVisit(Visit)
+	 */
+	@Test
+	public void purgeVisit_shouldEraseTheVisitFromTheDatabase() {
+		VisitService vs = Context.getVisitService();
+		Integer originalSize = vs.getVisits(null, null, null, null, null, null, null, null, null, true, true).size();
+		Visit visit = Context.getVisitService().getVisit(1);
+		vs.purgeVisit(visit);
```

Comment:
add a newline. this way the action we want to test `vs.purgeVisit` is clearly visible when you first look at the test.

like in the arrange, act, assert I showed you before which makes it easier to read.